code/datums/gamemode/factions/syndicate/nukeops.dm

```diff
@@ -26,6 +26,83 @@
 		. += "<BR>The operatives bought:<BR>"
 		for(var/entry in faction_scoreboard_data)
 			. += "[entry]<BR>"
+	var/diskdat = null
+	var/bombdat = null
+	var/opkilled = 0
+	var/oparrested = 0
+	var/alloparrested = 0
+	//var/nukedpenalty = 1000
+	for(var/datum/role/R in members)
+		var/datum/mind/M = R.antag
+		if(!M || !M.current)
+			opkilled++
+			continue
+		var/turf/T = M.current.loc
+		if(T && istype(T.loc, /area/security/brig))
+			oparrested++
```

Comment:
Does this factor in the arrest area of the escape shuttle? I believe there was a way to do this by checking if you're in the escape shuttle, then checking the tile they're on (since these tiles are indestructible, normally).

EDIT: It's pre-implemented in https://github.com/vgstation-coders/vgstation13/blob/6a74d7cff03ffa10df4661e373c0fdafa715fe56/code/datums/gamemode/objectives/escape.dm#L20

Reply:
just made it now